http_check/datadog_checks/http_check/utils.py

```diff
@@ -2,6 +2,13 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import os
+import platform
+import sys
+
+EMBEDDED_DIR = 'embedded'
+
+if platform.system() == 'Windows':
+    EMBEDDED_DIR += str(sys.version_info[0])
```

Comment:
Would be great to have a similar test for windows to test this case: https://github.com/DataDog/integrations-core/blob/5a326eae64876a8344ff8ef995604dd73793e9e7/http_check/tests/test_utils.py#L36-L57